cmd/notation/cert.go

```diff
@@ -3,42 +3,55 @@ package main
 import (
 	"errors"
 	"fmt"
-	"os"
-	"path/filepath"
-
-	"github.com/notaryproject/notation-core-go/x509"
-	"github.com/notaryproject/notation-go/config"
-	"github.com/notaryproject/notation/internal/ioutil"
-	"github.com/notaryproject/notation/internal/slices"
-	"github.com/notaryproject/notation/pkg/configutil"
+	"strings"
+
+	corex509 "github.com/notaryproject/notation-core-go/x509"
+	"github.com/notaryproject/notation-go/dir"
+	"github.com/notaryproject/notation-go/verification"
+	"github.com/notaryproject/notation/internal/truststore"
 	"github.com/spf13/cobra"
 )
 
 type certAddOpts struct {
-	path string
-	name string
+	storeType  string
+	namedStore string
+	path       []string
+}
+
+type certListOpts struct {
+	storeType  string
+	namedStore string
+}
+
+type certShowOpts struct {
+	storeType  string
+	namedStore string
+	cert       string
 }
 
-type certRemoveOpts struct {
-	names []string
+type certDeleteOpts struct {
+	storeType  string
+	namedStore string
+	cert       string
+	all        bool
+	confirmed  bool
 }
 
 type certGenerateTestOpts struct {
 	name      string
 	bits      int
 	trust     bool
-	hosts     []string
 	isDefault bool
 }
 
 func certCommand() *cobra.Command {
```

Comment:
The cert command is getting complex. Can we follow the same pattern as `oras` like `cert/add.go`, `cert, show.go`, etc..